blockchain/types/anchoring_data.go

```diff
@@ -31,6 +31,16 @@ type AnchoringData struct {
 	Data []byte
 }
 
+// AnchoringDataInternalNoType is an old anchoring type that does not support an data type.
+type AnchoringDataInternalNoType struct {
```

Comment:
How changing the name to `AnchoringDataInternalDefault` if it is used as a fallback type.

Reply:
Actually, the default is type0.